### PR TITLE
tools: Add TASH CMD auto test tool

### PR DESCRIPTION
**Pull request by KimDongEon**
This tool is a tool that automatically executes TASH commands.

[Install]
$ apt install python2.7
$ pip install pyserial

[How to Use]
$ python cmdautotest.py serial_port

ex)
$ python cmdautotest.py /dev/ttyUSB1

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0bc0558cac3fbff0539ecf852071ccd61da42335] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0bc0558cac3fbff0539ecf852071ccd61da42335] - Code Rule Check OK.

File: tools/autotest/auto-test.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
+from collections import OrderedDict
```
Comment (sunghan-chang) [inline tools/autotest/auto-test.py]: 1. Needs Samsung License.
2. Could you add the README to say how to use?
3. I don't know why this is necessary. What kind test does this do?

File: tools/autotest/auto-test.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+
+def cmd_date():
+    result = execute_cmd("cd date", "", PORT, 5)
+    return result
+
+def cmd_df():
+    data = []
+    data.append("Block  Number")
+
+    result = execute_cmd("df", data, PORT, 5);
```
Comment (junmin-kim) [inline tools/autotest/auto-test.py]: `;` is unnecessary in python, isn't it?

**Comment by junmin-kim** [comment]
There are a lot of PEP8 violation.
We can modify them by using lint

File: tools/autotest/auto-test.py (hunk 2 above)
Comment (KimDongEon) [reply tools/autotest/auto-test.py]: I will fix

**Comment by seinfra** [comment]
Target : [f697ea3a08bb40000d7340efd926f1ef9bf341da] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f697ea3a08bb40000d7340efd926f1ef9bf341da] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6b5cef9ada2972e1502dfcbc4091a3847b0381b0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6b5cef9ada2972e1502dfcbc4091a3847b0381b0] - Code Rule Check (C++) OK.

File: tools/autotest/auto-test.py (hunk 1 above)
Comment (KimDongEon) [reply tools/autotest/auto-test.py]: I will fix
1. Add Samsung License.
2. Add README file
3. This is used to quickly and automatically execute various commands when performing code coverage tests.

**Comment by seinfra** [comment]
Target : [5be20a6444c485b26f5407301e73b412facab73b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [5be20a6444c485b26f5407301e73b412facab73b] - Code Rule Check (C++) OK.

**Comment by chanijjani** [comment]
> $ apt isntall python2.7
--> install

> [How to Use]
> $ python auto-test.py serial_port
--> cmdautotest.py
> ex)
> $ python auto-test.py /dev/ttyUSB1
--> cmdautotest.py

File: tools/cmdautotest/cmdautotest.py [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+        except FileNotFoundError:
+            print ("FILE Not Found Error")
+            time.sleep(2)
+            port.open()
+        except serial.serialutil.SerialException:
+            print ("serial.serialutil.SerialException")
+            time.sleep(2)
+            port.open()
+
+        print ("Port connect = ", port.isOpen())
```
[the remark this answers is not in the document]
Comment (KimDongEon) [reply tools/cmdautotest/cmdautotest.py]: This indentation is correct.

**Pull request merged**